src/push/Push.js

```diff
@@ -248,22 +248,24 @@ export default class Push {
             const wrapper = this._prepareNotification(id, options);
 
             /* Notification callbacks */
-            if (Util.isFunction(options.onShow))
-                notification.addEventListener('show', options.onShow);
+            if (Util.isFunction(notification.addEventListener)) {
+              if (Util.isFunction(options.onShow))
+                  notification.addEventListener('show', options.onShow);
 
-            if (Util.isFunction(options.onError))
-                notification.addEventListener('error', options.onError);
+              if (Util.isFunction(options.onError))
```

Comment:
Overall looks good, but it looks like a bunch of these changes are just adding indents? Or am I missing something?

Reply:
Yes, these are just indents caused by `if (Util.isFunction(notification.addEventListener))` that wraps these `if` statements.

Reply:
Ah, my bad. Approved.